addons/point_of_sale/static/src/js/models.js

```diff
@@ -1539,7 +1544,7 @@ export class Product extends PosModel {
                     return pricelist.id === rule.base_pricelist_id[0];
                 });
                 if (base_pricelist) {
-                    price = self.get_price(base_pricelist, quantity);
+                    price = self.get_price(base_pricelist, quantity, recurring=true);
```

Comment:
I'm sorry, could you please explain to me why it's not working and what I should do instead? I didn't encounter any errors with this, so I assumed it was working '^^

Reply:
Answered in discord, for history sake since I wasn't super clear in my comment:
named parameters are not supported by JS (there are workaround for that). So basically `recurring=true` is doing an assignment to the variable `recurring` and is returning `true` for the third parameter which is `extra_price`.
So it's the same as doing: 
```js
recurring = true;
self.get_price(base_pricelist, quantity, true);
```